ocfweb/component/lab_status.py

```diff
@@ -17,15 +16,8 @@
 @periodic(60, ttl=86400)
 def get_lab_status():
     """Get the front page banner message from the default location."""
-    try:
-        with open('/home/s/st/staff/lab_status.yaml') as f:
-            tree = yaml.safe_load(f)
-    except IOError:
-        tree = yaml.safe_load(
-            requests.get(
-                'https://www.ocf.berkeley.edu/~staff/lab_status.yaml',
-            ).text,
-        )
+    with open('/etc/ocf/lab_status.yaml') as f:
+        tree = yaml.safe_load(f)
```

Comment:
this should also probably fail with the IOError, in case there's a problem mounting the directory into the container

Reply:
I was thinking that we'd delete the lab_status.yaml file from ~staff once this is merged

Reply:
keeping the old thing would actually be nice as a backup, but deleting it is fine too.

Reply:
Yeah, that is true. I was thinking it might lead to confusion in the future though, on whether you have to change the file in etc, the one in ~staff, or both.

Reply:
We shouldn't have a "fallback" option unless there's a way of notifying us that the fallback path was taken, otherwise it "fails" silently. In that case, it should be part of ocflib (and used for all `etc` configs), not just this one.